supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -601,6 +601,24 @@ Use all lower case, unless used in a title or at the beginning of a sentence.
 
 *See also*:
 
+[discrete]
+[[sos-report]]
+==== image:images/yes.png[yes] sos report (noun)
+*Description*: An _sos report_ is an archive of system information, including configuration and diagnostic information.
+Red Hat Support engineers review sos reports when troubleshooting system problems.
+The command to generate an sos report depends on the RHEL version:
+
+* In RHEL 8 and later, run the `sos report` command (that is, the `sos` command with the `report` argument).
+* In RHEL 7 and earlier, run the `sosreport` command.
+
+NOTE: The `sos` command is an acronym for "son of sysreport", and the official pronunciation is "ess-oh-ess". Therefore, use the indefinite article "an".
+
+*Use it*: yes
+
+*Incorrect forms*: sosreport (correct only for the command name in RHEL 7 and earlier), SOSreport, SOS report, or any other variation
```

Comment:
```suggestion
*Incorrect forms*: sosreport (incorrect except when referring to the command name in RHEL 7 and earlier), SOSreport, SOS report, or any other variation
```
Maybe better to say that it's always incorrect except for this one usage case? WDYt?

Reply:
Great point, @bburt-rh -- I've made that change, thanks!